plugins/modules/cloud/misc/proxmox_kvm.py

```diff
@@ -853,6 +955,12 @@ def create_vm(module, proxmox, vmid, newid, node, name, memory, cpu, cores, sock
         kwargs['numa'] = kwargs['numa_enabled']
         del kwargs['numa_enabled']
 
+    # PVE api expects strings for the following params
+    if module.params['nameservers']:
+        kwargs['nameserver'] = ' '.join(module.param['nameservers'])
+    if module.params['searchdomains']:
+        kwargs['searchdomain'] = ' '.join(module.param['searchdomains'])
```

Comment:
Since `kwargs` is passed on, I guess we have to `pop` the other names, i.e.
```suggestion
        kwargs['searchdomain'] = ' '.join(module.param['searchdomains'])
    kwargs.pop('nameservers')
    kwargs.pop('searchdomain')
```

Reply:
Sorry, forgot an `s`, see my next suggestion.